js/js.go

```diff
@@ -73,7 +73,7 @@ func (o *Object) Uint64() uint64 { return o.object.Uint64() }
 // Float returns the object converted to float64 according to JavaScript type conversions (parseFloat).
 func (o *Object) Float() float64 { return o.object.Float() }
 
-// Interface returns the object converted to interface{}. See GopherJS' README for details.
+// Interface returns the object converted to interface{}. See table in package comment for details.
```

Comment:
I'm worried that package documentation means all of the documentation, it's not very specific.

I wanted to refer to the package comment - the comment on package declaration - which is just the top-most section of package documentation, where the table is contained.

From https://blog.golang.org/godoc-documenting-go-code:

> **Comments on package declarations** should provide **general package documentation**. These comments can be short, like the sort package's brief description:
>
> ```Go
> // Package sort provides primitives for sorting slices and user-defined
> // collections.
> package sort
> ```
>
> They can also be detailed like the gob package's overview. That package uses another convention for packages that need large amounts of introductory documentation: the **package comment** is placed in its own file, doc.go, which contains only those comments and a package clause.
>
> When writing **package comments** of any size, keep in mind that their first sentence will appear in godoc's package list.

Emphasized relevant phrases used. What do you think?